randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -419,6 +436,8 @@ Extra - asset_id: 11
           Any of the following:
               Morph Ball Bombs or Power Bombs ≥ 4
               Power Bombs and Screw Attack
+              # Shinespark across: https://youtu.be/27WoKKQMn7w
```

Comment:
Not quite sure how to handle this one. Someone accidentally landing a yt vid with the title "sexmaster" doesn't really sit right with me. 
Might be better to just have "Sexmaster shinespark" as the comment. Dunno whether that's already too nsfw, but it's also how the trick is named, so might as well be upfront about it. IDK

Reply:
Might need a third opinion. I don't think the word sex is particularly NSFW, but leaving a comment with the title is not any better. The only way to completely circumvent it is to host a new video without that title.

Reply:
The issue about a new video with a "more safe" title, is that the name sexmaster is how it's referred to in the community. Meaning that if someone would research more about the trick (i.e. to look for a more indepth tutorial), they'd stumble upon the name anyway

Reply:
In that case, I think we should just ignore it. We're using the speedrunning wiki as our source for names and strats. We use what they use.

Reply:
in that case, can you change the comment to "sexmaster shinespark"?

Reply:
updated